src/js/themes/base.js

```diff
@@ -213,7 +213,10 @@ export const generate = (baseSpacing = 24, scale = 6) => {
           color: 'focus',
         },
       },
-      font: { ...fontSizing(0) },
+      font: { 
+        ...fontSizing(0), 
+        face: undefined, 
```

Comment:
I think it would be more consistent with the rest of this object if we commented this line out.